sql/src/test/java/io/crate/analyze/GroupByAnalyzerTest.java

```diff
@@ -344,13 +345,23 @@ public void testGroupWithInvalidNullLiteral() throws Exception {
     @Test
     public void testGroupByHaving() throws Exception {
         AnalyzedRelation relation = analyze("select sum(floats) from users group by name having name like 'Slartibart%'");
-        assertThat(relation.having().query(), isFunction("op_like"));
+        assertThat(relation.having().query(), isFunction(LikeOperators.OP_LIKE));
         Function havingFunction = (Function) relation.having().query();
         assertThat(havingFunction.arguments().size(), is(2));
         assertThat(havingFunction.arguments().get(0), isReference("name"));
         assertThat(havingFunction.arguments().get(1), isLiteral("Slartibart%"));
     }
 
+    @Test
+    public void testCaseInsensitiveGroupByHaving() throws Exception {
```

Comment:
I think the test names could still be further improved. E.g. `test_ilike_can_be_used_in_having_clause` (which makes me wonder why this is a special case and why we'd test it?)

We're not testing all operator placement permutations.